ydb/tools/ydbd_slice/nodes.py

```diff
@@ -107,8 +107,8 @@ def _copy_between_nodes(self, hub, hub_path, hosts, remote_path):
             if self._dry_run:
                 continue
             cmd = [
-                "ssh", dst, "-A", "sudo", "rsync", "-avqW", "--del", "--no-o", "--no-g",
-                "--rsh='ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -l %s'" % os.getenv("USER"),
+                "ssh", dst, '-o', 'StrictHostKeyChecking=no', '-o', 'UserKnownHostsFile=/dev/null',  "-A", "sudo", "rsync", "-avqW", "--del", 
```

Comment:
The tool was hung because of prompt`Are you sure you want to continue connecting (yes/no/[fingerprint])?`